datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 # CHANGELOG - {integration_name}
 
-{changelog_body}
```

Comment:
We have different changelog formats for ints-core and extras/marketplace.

The template used for y'all is [here](https://github.com/DataDog/integrations-core/blob/f987b7f44e7c8f29d2db8608dbcd1e916e584445/datadog_checks_dev/datadog_checks/dev/tooling/commands/create.py#L86). Please change that instead.